usecase/authz_proxyd.go

```diff
@@ -191,6 +193,19 @@ func newAuthzD(cfg config.Config) (service.Authorizationd, error) {
 				),
 			),
 		}
+	} else {
```

Comment:
In athenz-authorizer, access token is enabled by default.
https://github.com/yahoojapan/athenz-authorizer/blob/d896d3edec8f0d2abd90d20ff3ffd84abb2e702c/option.go#L38
Therefore, the existing code always set access token to enable.
Added an `else` to the `if` to make it properly disable.
The conditions of the JWKD are also unified here.